lib/code-manager.js

```diff
@@ -126,14 +144,17 @@ export function getRegexString() {
   return regexString;
 }
 
-export function getBreakpoints() {
-  const buffer = store.editor.getBuffer();
-  const breakpoints = [buffer.getFirstPosition()];
+export function getBreakpoints(editor: atom$TextEditor) {
+  const buffer = editor.getBuffer();
+  const breakpoints = [new Point(0, 0)];
 
-  const regexString = getRegexString(store.editor);
+  const regexString = getRegexString(editor);
   if (regexString) {
     const regex = new RegExp(regexString, "g");
-    buffer.scan(regex, ({ range }) => breakpoints.push(range.start));
+    buffer.scan(regex, ({ range }) => {
+      breakpoints.push(range.start);
+      return; // Flow does not want an integer returned by .push(range.start)
```

Comment:
I think this line can be removed. When using arrow functions with curly brackets it doesn't automatically return.